### DISCUSSION TITLE
pythonanywhere.com

### DISCUSSION THREAD

**Discussion by trinkey** [catppuccin/catppuccin#2339] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

usercss (aka hell)

### Any additional comments?

ill be able to start working on sunday probably although if anyone else wants to start/do it go ahead just lmk

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by NeonGamerBot-QK** [comment]
![](https://cdn.discordapp.com/emojis/986539896409841684.png?v=1&size=16)  https://github.com/NeonGamerBot-QK/catppuccin-userstyles/tree/feat/pythonanywhere.com WIP

**Comment by trinkey** [comment]
i already started it and its almost done except i lost all motivation
https://github.com/catppuccin/userstyles/pull/776

feel free to finish it if you want

**Comment by NeonGamerBot-QK** [comment]
uh i might

**Comment by sgoudham** [comment]
I've closed the original PR. Please raise a new PR if you're interested in seeing this website themed.
If you're using some of the contributions from the original author then co-authoring seems like the best way forward.

**Comment by AlwaysNur** [comment]
We now have this [here](https://github.com/catppuccin/userstyles/tree/main/styles/pythonanywhere)
